summary_workflows/quantification/nextflow.config

```diff
@@ -30,26 +30,51 @@ profiles {
       // set time zone for running docker containers
       docker.runOptions = '--user \$(id -u):\$(id -g) -e TZ="\$([ -z \\"\$TZ\\"] && cat /etc/timezone || echo \\"\$TZ\\")"'
   }
+
+  slurm {
+      singularity.enabled = true
+      singularity.autoMounts = true
+      singularity.cacheDir = "${HOME}/.singularity/cache/library"
+      singularity.libraryDir = "${HOME}/.singularity/cache/library"
+
+      process {
+              executor = 'slurm'
+              memory = '4 GB'
+              cpus = '1'
+
+              withName: validation{
+                        container = "docker.io/cjh4zavolab/q_validation:1.0"
```

Comment:
because I needed it available online, as I can't build singularity containers from Dockerfiles, only pull them. As I don't have push access to the APAeval dockerhub it's on my own dockerhub. When we have a final version we'll have to ask someone to push the images to the APAeval dockerhub